client/src/components/Auth/LoginForm.tsx

```diff
@@ -74,7 +74,7 @@ function LoginForm({ onSubmit }: TLoginFormProps) {
               },
               maxLength: {
                 value: 40,
-                message: 'Password must be less than 40 characters'
+                message: 'Password must be less than 128 characters'
```

Comment:
Minor note: if 128 is the max length, then the password must be "less than or equal to", not "less than" 128 characters.